### ISSUE TITLE
Cut an official release of appservices with nimbus

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/application-services#3699]
┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/SYNC-1884)
┆Story Points: 3
┆Epic: Fenix Client SDK
┆Sprint: SYNC - end 2020-11-20

**Comment by rfk** [comment]
The bulk of this work is done in https://github.com/mozilla/application-services/pull/3701.

@travis79 are there pending nimbus PRs that we should merge before cutting a release here? I'm thinking specifically of [global opt-out](https://github.com/mozilla/nimbus-sdk/pull/51) and [schema version checking](https://github.com/mozilla/nimbus-sdk/pull/49), but I think we could safely publish those in a follow-up release if we want to get something out the door to unblock the [main a-c PR]9https://github.com/mozilla-mobile/android-components/pull/8261). WDYT?

**Comment by travis79** [comment]
While it would be nice to have the global opt-out and the versioning, I'm fine with landing those after a release to unblock the android component landing.

**Comment by jhugman** [comment]
@travis79 — I'd love to be able to land global opt out before the android-component lands.

**Comment by rfk** [comment]
Version `v67.0.0` has been tagged and will include the new nimbus component. Consumers should take a dependency on `org.mozilla.appservices:nimbus`.

**Comment by rfk** [comment]
>  I'd love to be able to land global opt out before the android-component lands.

Noting for completeness, it does include the global opt-out functionality.
